src/main/sh/mommy

```diff
@@ -358,6 +359,54 @@ done
 
 shift "$((OPTIND - 1))"
 
+# do the symlinking
+if [ -n "$opt_rename" ]; then
+    if [ $# -lt 1 ]; then
+        die "mommy expected a file path to rename"
+    fi
+    echo "mommy is locating her shell script"
+    findoldpath=0
+    oldpath="$(whereis mommy | sed 's/^mommy://')"
+    
+    if is_blank "$oldpath"; then
+        echo "Mommy's shell script is not in path, mommy's gonna search your whole disk now"
+        echo "Warning: Mommy's search could be extremely slow on hard drives"
+        findoldpath=1
+        mommyshellscript="$(find / -name mommy -type f 2>/dev/null)"
+        mommymandocs="$(find / -name mommy.1.gz -type f 2>/dev/null)"
```

Comment:
i really don't think this is a good idea. there are so many ways in which this can go wrong. you might be entering network drives and then trying to create symlinks on a remote system, or someone might have a game or personal file called `mommy`, or... et cetera.

instead of guessing, please just let the program fail and give the user an appropriate warning.